Kickstarter-iOS/Views/Controllers/PledgeViewController.swift

```diff
@@ -479,8 +478,9 @@ final class PledgeViewController: UIViewController,
     self.present(paymentAuthorizationViewController, animated: true)
   }
 
-  private func goToRiskMessagingModal() {
-    let viewController = RiskMessagingViewController()
+  private func goToRiskMessagingModal(isApplePay: Bool) {
+    self.viewModel.inputs.configureRiskMessagingModal(isApplePay: isApplePay)
+    let viewController = RiskMessagingViewController(viewModel: self.viewModel)
```

Comment:
Yea not 100% about this approach. Usually, view models are contained within their own view controllers. You can pass _in_ data that can be used to create a new view model within `RiskMessagingViewController` (ie. `RiskMessagingViewModel`), but passing in `PledgeViewModel` to `RiskMessageViewController` directly seems a bit hacky to me.

I would suggest creating a new data type struct/enum similar to `ManagePledgeViewParamConfigData` that can be passed around, I see a lot of examples like that in the app.

A view model should be privately held by its view not accessible outside of it, separation of concerns.